apps/vanilla/src/pages/index.html

```diff
@@ -104,9 +104,9 @@
       <!-- Filter -->
       <section class="filters">
         <section class="filters__form-wrapper form-wrapper">
-          <div class="font-title">Filter</div>
-          <form class="form-wrapper__form-control form-control">
-            <section class="form-control__field-style field-style">
+          <h2 class="font-title">Filter</h2>
```

Comment:
Add missing space for filter title
```suggestion
          <h2 class="font-title">Filters</h2>
```
![image](https://github.com/saritasa-nest/camp-js-2024-dat-ngo/assets/108647643/2f1610b8-daf7-44b7-89d7-532e1bd19e6b)